packages/polyfills-loader/package.json

```diff
@@ -9,16 +9,22 @@
   "publishConfig": {
     "access": "public"
   },
+  "scripts": {
+    "start": "http-server -o"
```

Comment:
what does this do? I don't see an index.html? should there be a demo folder or so?

Reply:
I had set up a demo for testing, but realized it wouldn't work without module resolution. So I removed it again. Removed the scripts now too.